graspologic/cluster/autogmm.py

```diff
@@ -657,6 +676,13 @@ def _process_paramgrid(paramgrid):
         Each dict has the keys 'affinity', 'covariance_type', 'linkage',
         'n_components', and 'random_state'
 
+    run_multiple_init : bool
+        If True, run additional n_init k_means initializations.
+        Else, no multiple k-means initialization is performed.
+
+    n_init : int, defaults to 1.
```

Comment:
Is it default to 1 or None? It says None at line 231?

Reply:
It is None, I will promptly edit it